tests/src/test/scala/com/nvidia/spark/rapids/CollectionOpSuite.scala

```diff
@@ -19,22 +19,18 @@ package com.nvidia.spark.rapids
 import org.apache.spark.sql.functions.map_concat
 
 class CollectionOpSuite extends SparkQueryCompareTestSuite {
-  testGpuFallback(
-    "MapConcat with Array keys fall back",
-    "ProjectExec",
-    ArrayKeyMapDF, 
-    execsAllowedNonGpu = Seq("ProjectExec", "ShuffleExchangeExec")) {
+  testSparkResultsAreEqual(
+    "MapConcat with Array keys",
+    ArrayKeyMapDF) {
     frame => {
       import frame.sparkSession.implicits._
       frame.select(map_concat($"col1", $"col2"))
     }
   }
 
-  testGpuFallback(
-    "MapConcat with Struct keys fall back",
```

Comment:
Should there be a corresponding test for `MapType` as well?

Reply:
Spark doesn't support `MapType` as a key type. If you use `Map` as a key, the Spark catalyst will throw an error.